evacuees/src/API/EMBC.Tests.Integration.Registrants.API/Evacuations/EvacuationTests.cs

```diff
@@ -79,6 +79,8 @@ public async Task CanCreateEvacuation()
             var createdEvacuationFile = await evacuationManager.GetEvacuation(testUserId, newEssFileNumber);
 
             createdEvacuationFile.ShouldNotBeNull();
+
+            await evacuationManager.DeleteEvacuation(testUserId, newEssFileNumber);
```

Comment:
Shouldly has the ability to test for exceptions, or check that the evacuation is no longer accessible